include/zephyr/bluetooth/conn.h

```diff
@@ -640,6 +640,10 @@ struct bt_conn_le_cs_subevent_result {
 		/** Number of CS steps in the subevent.
 		 */
 		uint8_t num_steps_reported;
+		/** Step number, on which the subevent was aborted
+		 *  255 if subevent was not aborted
```

Comment:
```suggestion
		 *  255 if subevent_done_status is @ref BT_CONN_LE_CS_SUBEVENT_COMPLETE
```

we can consider removing BT_CONN_LE_CS_SUBEVENT_INCOMPLETE since this is no longer exposed to the user